### PR TITLE
Classification: default semilar threshold

### PR DESCRIPTION
**Pull request by simonzachau**

### CODE REVIEW

File: app/classification.js [hunk 1]
```diff
@@ -79,8 +84,30 @@ module.exports.getSemilarType = function(word) {
...
+      }, {type: null, similarity: config.semilarAlgorithmThreshold})
     }).catch(() => {
       return {type: null, similarity: 0};
     });
 };
+
+// return string without nouns, adjectives, or adverbs
+exports.filterMeaningfulVerb = function (relation) {
+  let words = relation.split(' ');
+  let customWordsToExclude = ['was', 'were', 'to', 'be', 'am', 'is', 'are', 'also', 'then', 'had', 'has', 'have'];
```
Comment (Henni) [inline app/classification.js:96]: did you come up with this yourself or is there maybe some list we could use in any of our libraries?

File: app/utils.js [hunk 2]
```diff
@@ -78,3 +78,15 @@ module.exports.downloadAndUnzipFile = function(url, outputDir){
       });
   });
 };
+
+exports.array2String = function (array) {
+  return array.reduce((acc, word) => acc + ' ' + word, '').slice(1);
+};
```
Comment (Henni) [inline app/utils.js]: should be replace with `array.join()` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

**Review by Henni — CHANGES_REQUESTED** [review]

File: app/utils.js [hunk 3]
```diff
@@ -78,3 +78,15 @@ module.exports.downloadAndUnzipFile = function(url, outputDir){
...
+
+exports.removeArrayElements = function(array, elementsToBeRemoved) {
+  elementsToBeRemoved.forEach(element => {
+    var i = array.indexOf(element);
+    if (i != -1) array.splice(i, 1);
+  });
+  return array;
+};
```
Comment (Henni) [inline app/utils.js:88]: could be rewritten with a pretty filter function, but fine for now...

File: app/classification.js (hunk 1 above)
Comment (simonzachau) [reply app/classification.js:96]: I came up with it myself. If you find an official list, feel free to include it!

Comment (simonzachau) [reply app/classification.js:96]: e.g. http://xpo6.com/list-of-english-stop-words, but the list is so long it'd be cleaner to read in a file with all the words.

**Review by Henni — APPROVED** [review]

File: app/utils.js (hunk 3 above)
Comment (simonzachau) [reply app/utils.js:88]: taken care of in #62

**Pull request merged**